data/madlib/templates.json

```diff
@@ -0,0 +1,75 @@
+		],
+		"libra": [
+			"Some people are so focused on their own {{noun}}, that they refuse to {{present_verb}} or take on board sane advice. This is something you might need to {{present_verb}}, Libra. You'll soon realize it's a futile {{noun}}. Don't bother even {{ing_verb}}. Instead, just leave it {{adjective}}. This is one of those times when allowing {{person}} to experience their {{noun}} could be the only thing that pushes them to {{present_verb}}."
+		],
+		"pisces": [
+			"It won't {{present_verb}} to get into a {{noun}} with a friend or {{person}}, as no-one will {{present_verb}}, and it could {{present_verb}} a relationship that is otherwise {{adjective}}. The {{present_verb}} to {{present_verb}} a {{noun}} just for the sake of it, might {{present_verb}} you or another to {{present_verb}} things you may {{present_verb}}. Yet within a day or so, you can {{present_verb}} what all the {{noun}} was about, Pisces. In this regard, {{ing_verb}} your {{adjective}} {{asset}} over logic could {{present_verb}} you."
+		],
+		"sagittarius": [
+			"The {{planet}}'s {{adjective}} with {{adjective}} {{planet}}, could be a {{noun}} that {{present_verb}} you to try something {{adjective}}. This should appeal to your {{asset}} and your {{adjective}} to explore the unknown. You may be {{adjective}} by what's going on in your local {{noun}}, Archer. Or be {{ing_verb}} for {{noun}} or {{noun}}. If a new {{noun}} or {{noun}} clicks, why not give it a go. It might be {{adjective}} for you."
+		],
+		"scorpio": [
+			"{{ing_verb}} can be {{adjective}}, and you might {{adverb}} {{present_verb}} of the {{noun}} going around in your {{asset}}. One way to {{present_verb}} this could be to get {{adjective}} and focus on {{ing_verb}} your {{asset}}, as this can help earth {{adjective}} energy and allow your {{asset}} to {{present_verb}}. It helps to avoid {{adjective}} moves, and to {{present_verb}} before any {{adjective}} {{noun}}."
+		],
+		"taurus": [
+			"You might wonder why your thoughts about {{person}} are so {{adjective}}, and what you could do to stop this. It's {{adverb}} because {{planet}} is still linked to {{planet}}, and thankfully this {{adjective}} is {{ing_verb}}. This may not help you now, when you'd like nothing more than to {{present_verb}}. Best to {{present_verb}} and forget about it ol' chap."
+		],
+		"virgo": [
+			"Something you want could be {{ing_verb}} way too much bandwidth, and causing you to {{present_verb}}, Virgo. In a few days, you may not {{present_verb}} the same way and might have little {{noun}} in it. This all or nothing approach is typical of the current {{planets}}/{{planets}} angle, so it's worth {{ing_verb}} about this in advance. If you go all out and {{present_verb}}, you may soon wonder why you bothered."
+		],
+		"al": [
```

Comment:
Doesn't need to be addressed here but it might be fun to look at working templates into these.